packages/access-api/src/types/provisions.ts

```diff
@@ -1,23 +1,21 @@
 import * as Ucanto from '@ucanto/interface'
 import { ProviderAdd } from '@web3-storage/capabilities/src/types'
 
-export type AlphaStorageProvider = 'did:web:web3.storage:providers:w3up-alpha'
-
 /**
  * action which results in provisionment of a space consuming a storage provider
  */
 export interface Provision<ServiceDID extends Ucanto.DID<'web'>> {
   invocation: Ucanto.Invocation<ProviderAdd>
   space: Ucanto.DID<'key'>
   account: Ucanto.DID<'mailto'>
-  provider: AlphaStorageProvider | ServiceDID
+  provider: ServiceDID
 }
 
 /**
  * stores instances of a storage provider being consumed by a consumer
  */
 export interface ProvisionsStorage<ServiceDID extends Ucanto.DID<'web'>> {
-  service: ServiceDID
+  services: ServiceDID[]
```

Comment:
Nit: I'd remove these from the interface entirely. When I was designing provider protocol I was assuming that we would have `provider/publish` capability which would allow you to create a provider and consequently allow others to add it if they received `consumer/add` delegation from you. I realize we're far from there, but it still might be good idea to align on it. Implementation could still have such a field.